.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,22 +2,31 @@
 name: Bug report
 about: Create a bug report
 title: ''
-labels: ''
+labels: bug
 assignees: ''
 
 ---
 
-### Description
+**Description:**
+A clear and concise description of what the bug is.
 
-<!--
-* Please share short description of the problem
-  -->
+**Expected behavior:**
+A description of what you expected to happen.
 
-### Details
+**Actual behavior:**
+A description of what is actually happening.
```

Comment:
I think Expected and Actual behavior fields should be placed under repro steps. It looks more logical to me.

Reply:
+1